code/dictation.py

```diff
@@ -47,10 +68,34 @@ def capture_to_words(m):
     for item in m:
         words.extend(
             actions.user.replace_phrases(actions.dictate.parse_words(item))
-            if isinstance(item, grammar.vm.Phrase) else
-            item.split(" "))
+            if isinstance(item, grammar.vm.Phrase) else item)
```

Comment:
Note I changed this to be in consistent with @rntz suggestion and the behavior below.